libraries/chain/controller.cpp

```diff
@@ -1442,7 +1442,11 @@ struct controller_impl {
                }
 
                ++bb._pending_block_header_state.security_group.version;
-               bb._pending_block_header_state.security_group.participants = std::move(gpo.proposed_security_group_participants);
+               bb._pending_block_header_state.security_group.participants = {
+                  gpo.proposed_security_group_participants.begin(),
+                  gpo.proposed_security_group_participants.end(),
+                  bb._pending_block_header_state.security_group.participants.key_comp(),
+                  bb._pending_block_header_state.security_group.participants.get_allocator()};
```

Comment:
please correct me if I wrong, but we do not assign custom comparator or allocator anywhere, so constructor supposed to instantiate one from template parameter and we don't need to make here a copy from existing comparator and allocator

Reply:
yes, this should just use begin and end iterator